vars.sample

```diff
@@ -50,10 +50,20 @@ ORACLE_TABLESPACES_DIR=/home/oracle/app/oracle/oradata
 # }}}
 
 # Only needed for ETL
+ETL_SCRIPTS=/path/to/transmart-data/samples/postgres
+export PATH=$ETL_SCRIPTS:$PATH
+
+# export KETTLE_HOME=$HOME ## kettle.properties file is expected in .kettle subdirectory of KETTLE_HOME
+
 # End with /
-KETTLE_JOBS_PSQL=/path/to/transmart-ETL/Postgres/GPL-1.0/Kettle/Kettle-ETL/
+KETTLE_JOBS_PSQL=/path/to/tranSMART-ETL/Postgres/GPL-1.0/Kettle/Kettle-ETL/
+R_JOBS_PSQL=/path/to/tranSMART-ETL/Postgres/GPL-1.0/R
```

Comment:
You might as well define only the path to `tranSMART-ETL` then, with a mechanism that ensures backwards compatibility so that people that define only `KETTLE_JOBS_PSQL` don't get their setup broken.